stdlib/3/typing.pyi

```diff
@@ -317,7 +330,10 @@ class MutableSet(AbstractSet[_T], Generic[_T]):
     def __ixor__(self, s: AbstractSet[_S]) -> MutableSet[Union[_T, _S]]: ...
     def __isub__(self, s: AbstractSet[Any]) -> MutableSet[_T]: ...
 
-class MappingView(Sized):
+class MappingView:
+    def __len__(self) -> int: ...
+    # Implement Sized (but don't have it as a base class).
+    @abstractmethod
```

Comment:
Again, duplicate implementation of `__len__`.

Reply:
Ditto.